.github/workflows/build.yml

```diff
@@ -0,0 +1,113 @@
+name: Scala tests with coverage
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    paths-ignore: ['**.md']
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
```

Comment:
In Travis we had to specify the Scala version, does GHA know to just default to what Sam uses?

Reply:
I believe so? If not, it is dictated by the coursier/cache-action, which caches at least sbt.